benchmarks/benchmark/tools/profile-generator/container/benchmark_serving.py

```diff
@@ -324,20 +339,52 @@ def print_metrics(metrics: List[str], duration: float, backend: str):
 
   all_metric_results = {}
 
-  for metric in metrics:
+  for metric_alias, metric in metrics.items():
     print("Metric Name: %s" % (metric))
+    # Request refresh tokens
```

Comment:
Does authenticating once above not enough? What is the expiration on the token? Wondering why we need to refresh it since I assume this isn't too time consuming.

Reply:
This was old behavior but yes worth a look, this call was moved to before the `/metadata` call since it only needs to run once.